### PR TITLE
docs: clarify usage of `output` column generated by the exporter

### PR DESCRIPTION
**Pull request by gcroci2**

### CODE REVIEW

File: docs/getstarted.md [hunk 1]
```diff
@@ -391,6 +391,8 @@ output_test = pd.read_hdf(os.path.join("<output_folder_path>", "output_exporter.
 
 The dataframes contain `phase`, `epoch`, `entry`, `output`, `target`, and `loss` columns, and can be easily used to visualize the results.
 
+For classification tasks, the `output` column contains the results from a [softmax function](https://pytorch.org/docs/stable/generated/torch.nn.functional.softmax.html). This means each entry in the `output` column is a list with one element for each class. Each element represents the probability of that class occurring.
```
Comment (DaniBodor) [inline docs/getstarted.md:394]: I would simplify this a bit. Maybe also add a comment on what the order of the list is?

```suggestion
For classification tasks, the `output` column contains a list of probabilities that each class occurs. For more details, please see documentation on the [softmax function](https://pytorch.org/docs/stable/generated/torch.nn.functional.softmax.html)
```

File: tutorials/training.ipynb [hunk 2]
```diff
@@ -436,7 +432,11 @@
       "source": [
         "The dataframes contain `phase`, `epoch`, `entry`, `output`, `target`, and `loss` columns, and can be easily used to visualize the results.\n",
         "\n",
-        "For example, the loss across the epochs can be plotted for the training and the validation sets:\n"
+        "For classification tasks, as in the current tutorial, the `output` column contains the results from a [softmax function](https://pytorch.org/docs/stable/generated/torch.nn.functional.softmax.html). This means each entry in the `output` column is a list with one element for each class. Each element represents the probability of that class occurring.\n",
```
Comment (DaniBodor) [inline tutorials/training.ipynb:435]: see above

File: tutorials/training.ipynb [hunk 3]
```diff
@@ -436,7 +432,11 @@
...
+        "\n",
+        "Here specifically, the `output` column contains a list with two elements, respectively representing the predicted probabilities that the data point is 0 (first element of the list, representing non-binder class) and 1 (second element of the list, representing binder class).\n",
```
Comment (DaniBodor) [inline tutorials/training.ipynb:437]: why is 0 first and 1 second and not the other way around?

**Review by DaniBodor — APPROVED** [review]
Minor comment, but apart from that, go ahead

File: tutorials/training.ipynb (hunk 3 above)
Comment (gcroci2) [reply tutorials/training.ipynb]: They are ordered according to the `classes` attribute. I added a comment about that :)

**Pull request merged**